CMake/FindSphinx.cmake

```diff
@@ -73,6 +73,3 @@ endif()
 include(FindPackageHandleStandardArgs)
 
 find_package_handle_standard_args(Sphinx DEFAULT_MSG SPHINX_EXECUTABLE)
-if(PYTHON_EXECUTABLE) # prefix with python version used by build to avoid errors
```

Comment:
If you remove the build will fail somewhere else, @eile added it, maybe he remembers where he was having the error.